ansible/roles/stack-sunbird/templates/sunbird_cert-registry-service.env

```diff
@@ -1,3 +1,14 @@
 cert_service_base_url=http://cert-service:9000
 sunbird_es_host={{sunbird_es_host}}
 sunbird_es_port={{sunbird_es_port}}
+sunbird_cassandra_keyspace=sunbird
+sunbird_cassandra_username=cassandra
+sunbird_cassandra_password=password
+sunbird_cassandra_consistency_level=quorum
+{% if groups['cassandra-2'] is defined %}
+sunbird_cassandra_host={{groups['cassandra']|join(',')}}
+sunbird_cassandra_port=9042,9042,9042
```

Comment:
can't we just have a single port ?

Reply:
no, it supports like this only @rjshrjndrn